spec/acceptance_data.yml

```diff
@@ -0,0 +1,41 @@
+---
+-
+  desc: verify String handling
+  json: abc
+  string: abcS
+  signature: c4449120506d97975c67be69719a78e2
+-
+  desc: verify Array handling
+  json: [1, "a", 3]
+  string: 1N3NaSA
+  signature: cd1c43797d488d0f6c0d71537c64d30b
+-
+  desc: verify Array sorting
+  json: [3, null, 1, "1"]
+  string: _1N1S3NA
+  signature: 518e7bb17674f6acbb296845862a152d
+-
+  desc: verify hash like data structures
+  json: {"a": 1}
+  string: 1NaSAH
+  signature: 8cb44d69badda0f34b0bab6bb3e7fdbf
+-
+  desc: verify nested hash
+  json: {"a": {"c": nil, "2": 2 }}
```

Comment:
We couldn't pass this exact data structure into a `signature` function in other languages such as JavaScript, as `nil is not defined`. Would have to be `'nil'` or `null` I expect (but you'd get a different MD5 hash produced as a result).

Reply:
oops thus should be `null` as it is in line :14 
👍

Reply:
Do you need a `number` data type example in here?